src/main/java/org/gridsuite/study/server/networkmodificationtree/NetworkModificationNodeInfoRepositoryProxy.java

```diff
@@ -117,9 +123,10 @@ public UUID getSecurityAnalysisResultUuid(AbstractNode node) {
     }
 
     @Override
-    public void updateBuildStatus(AbstractNode node, BuildStatus buildStatus) {
+    public void updateBuildStatus(AbstractNode node, BuildStatus buildStatus, List<UUID> changedNodes) {
         NetworkModificationNode networkModificationNode = (NetworkModificationNode) node;
         networkModificationNode.setBuildStatus(buildStatus);
+        changedNodes.add(node.getId());
```

Comment:
To notify the front of which nodes has been modified

Reply:
Yes, but it is not clear, at first glance, why we have to pass a list of changed nodes in a method named updateBuildStatus. I prefer the way it was done before (node was added to the list of changed nodes after updateBuildStatus has been called, not in the method itself)

Reply:
After having merged the refactor PR, updateBuildStatus is now only overrided in ModelNodeInfoRepositoryProxy class (because only a ModelNode has a buildStatus), and so, adding the node in changedNodes list is now done only in this overrided method